### PR TITLE
mlt, bump version

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

**Comment by Begasus** [comment]
When launching "melt" in Terminal I see the following in syslog:
````
~> tail -f /var/log/syslog
KERN: package_daemon: [10823798507:    65] volume at "/Share/haikuports/media-libs/mlt/work-7.28.0/boot/system" unregistered
KERN: package_daemon: [10823798586:    65] root at "/Share/haikuports/media-libs/mlt/work-7.28.0/boot" unregistered
KERN: package_daemon: [10844848511:    87] CommitTransactionHandler::_ChangePackageActivation(): activating 0, deactivating 1 packages
KERN: unknown: [10844848763:    87] Volume::_ChangeActivation(): 0 new packages, 1 old packages
KERN: unknown: [10844853365:    87] package "mlt-7.28.0-1-x86_64.hpkg" deactivated
KERN: package_daemon: [10845349947:    87] Volume::_PackagesEntryRemoved("mlt-7.28.0-1-x86_64.hpkg")
KERN: package_daemon: [10851722075:    87] CommitTransactionHandler::_ChangePackageActivation(): activating 1, deactivating 0 packages
KERN: unknown: [10851723395:    87] Volume::_ChangeActivation(): 1 new packages, 0 old packages
KERN: unknown: [10851770806:    87] package "mlt-7.28.0-1-x86_64.hpkg" activated
KERN: package_daemon: [10852223231:    87] Volume::_PackagesEntryCreated("mlt-7.28.0-1-x86_64.hpkg")
KERN: runtime_loader: /boot/system/lib/mlt-7: Troubles reading ELF header
KERN: runtime_loader: /boot/system/lib: Troubles reading ELF header
KERN: vm_soft_fault: va 0x4a49808000 not covered by area in address space
KERN: vm_page_fault: vm_soft_fault returned error 'Bad address' on fault at 0x4a49808fd0, ip 0x4a49808fd0, write 0, user 1, exec 1, thread 0xa45a
````
The command lists all sorts of variables and then ends in the last line with: `Segmentation violation`

Need to check this on previous version, but I suspect it suffers from the same, maybe related to problems with KDenlive and Shotcut?

**Comment by Begasus** [comment]
Needs further testing, syslog is the same for the current version in the depot, but doesn't end in a `Segmentation violation` when launching in Terminal.

File: media-libs/mlt/patches/mlt-7.28.0.patchset [hunk 1]
```diff
@@ -33,23 +33,23 @@ index 22be72e..8134e27 100644
...
 --- a/src/modules/plusgpl/CMakeLists.txt
 +++ b/src/modules/plusgpl/CMakeLists.txt
 @@ -21,7 +21,7 @@ target_link_libraries(mltplusgpl PRIVATE mlt m Threads::Threads)
  
  if(WIN32)
    target_link_libraries(mltplusgpl PRIVATE ws2_32)
--elseif(UNIX AND NOT APPLE AND NOT ANDROID)
-+elseif(UNIX AND NOT APPLE AND NOT ANDROID AND NOT HAIKU)
+-elseif(UNIX AND NOT APPLE AND NOT ANDROID AND NOT ${CMAKE_SYSTEM_NAME} STREQUAL "OpenBSD")
++lseif(UNIX AND NOT APPLE AND NOT ANDROID AND NOT HAIKU AND NOT ${CMAKE_SYSTEM_NAME} STREQUAL "OpenBSD")
```
Comment (jmairboeck) [inline media-libs/mlt/patches/mlt-7.28.0.patchset:44]: Typo:

```suggestion
+elseif(UNIX AND NOT APPLE AND NOT ANDROID AND NOT HAIKU AND NOT ${CMAKE_SYSTEM_NAME} STREQUAL "OpenBSD")
```

Comment (Begasus) [reply media-libs/mlt/patches/mlt-7.28.0.patchset:44]: Me bumps head so hard! :) thanks!

**Comment by Begasus** [comment]
Looked like opencv was the problem here, disabling it fixed the error in Terminal, syslog entries are still the same.

**Pull request merged**